google-cloud-pubsub/lib/google/cloud/pubsub/v1/doc/google/pubsub/v1/pubsub.rb

```diff
@@ -37,7 +37,7 @@ class MessageStoragePolicy; end
       #     must not start with `"goog"`.
       # @!attribute [rw] labels
       #   @return [Hash{String => String}]
-      #     User labels.
+      #     See <a href="/pubsub/docs/labels"> Creating and managing labels</a>.
```

Comment:
There are still bad links in the documentation. Was this supposed to be addressed upstream?

Reply:
@dazuma ^

Reply:
Dammit, lemme ping the team...

Reply:
Update: The service team still has not chosen to update the public protos. We will hold here (and block the GA release of the pubsub library) until they do.

Reply:
@dazuma Should we revisit this? Can we replace `<a href="/pubsub/` with `<a href="https://cloud.google.com/pubsub/` using synth.py in the interim?

Reply:
I pinged the internal issue again. The protos are fixed internally; they just need to get released. I'd rather use this as a forcing function to get the internal release process fixed, rather than hack it on our end.

Reply:
Makes sense. Thanks.